src/default-xpring-client.ts

```diff
@@ -8,20 +8,31 @@ import GRPCNetworkClientWeb from './grpc-network-client.web'
 import { NetworkClient } from './network-client'
 import { GetTxResponse } from './generated/web/rpc/v1/tx_pb'
 import { GetFeeResponse } from './generated/web/rpc/v1/fee_pb'
-import { XRPDropsAmount } from './generated/web/rpc/v1/amount_pb'
+import {
+  AccountAddress,
+  CurrencyAmount,
+  XRPDropsAmount,
+} from './generated/web/rpc/v1/amount_pb'
 import isNode from './utils'
+import { Payment, Transaction } from './generated/web/rpc/v1/transaction_pb'
+import { AccountRoot } from './generated/web/rpc/v1/ledger_objects_pb'
 
 // TODO(keefertaylor): Re-enable this rule when this class is fully implemented.
 /* eslint-disable @typescript-eslint/require-await */
 /* eslint-disable @typescript-eslint/no-unused-vars */
 /* eslint-disable class-methods-use-this */
 
+/** A margin to pad the current ledger sequence with when submitting transactions. */
+const ledgerSequenceMargin = 10
```

Comment:
I recommend renaming `ledgerSequenceMargin` to `maxLedgerVersionOffset` for consistency with existing APIs, unless we have a good reason to introduce new terminology. https://xrpl.org/rippleapi-reference.html#transaction-instructions

Reply:
Done. Thanks for the tip.